deepchem/trans/transformers.py

```diff
@@ -1358,3 +1380,13 @@ def salt_pepper_noise(self, prob=0.05, salt=255, pepper=0):
     x[noise < (prob / 2)] = pepper
     x[noise > (1 - prob / 2)] = salt
     return x
+
+  def median_filter(self, size):
+    """ Calculates a multidimensional median filter
+    Parameters:
```

Comment:
Can you convert this to numpy docstring style too?

Reply:
Done :+1: